nixos/modules/services/system/dbus.nix

```diff
@@ -174,6 +174,10 @@ in
         permissions = "u+rx,g+rx,o-rx";
       };
 
+      system.switch.inhibitors = [
+        cfg.dbusPackage
```

Comment:
I'm pretty sure we already special case dbus.service (and a bunch of other services) in stc so that nixos-rebuild switch works without restarting these.

This breaks that no?